lib/make/makefile.bacmet

```diff
@@ -0,0 +1,209 @@
+# makefile.bacmet
+#
+# Library makefile to make running DIAMOND simpler against the bacmet database.
+#
+# Include the full path of this file in your Makefile and making databases and
+# running the aligner can be done via make. See documentation in in
+# the biomakefile Git repository: doc/makefile.md and documentation for
+# individual targets below.
+#
+# Author: diego.brambilla@tim.it
+
+SHELL := /bin/bash
+
+# *** Parameters ***
+
+# Override in your Makefile by setting a parameter *after* the row that
+# includes this file, see documentation in doc/makefile.md.
+
+# DIAMOND_DB_PATH sets the directory where to look for generic database files
+# like the protein RefSeq. The default is the current directory.
+DIAMOND_DB_PATH = .
+
+# Specify this when you want to point to a specific, non-generic, database.
+# Set DIAMOND_DB in a local Makefile to the
+# full path of the database file, without the ".dmnd" suffix.
+DIAMOND_DB =
+
+# deeparg.py specific macro: define minimum a.a. alignemnt length threshold
+ALENGTH = 37.5
+
+# Set this to suffixes corresponding to the types of files you wish to fetch with some
+# of the t$(DIAMOND_DB)ets.
+ACCEPTED_SUFFIXES =
+
+# DIAMOND_ALIGN_OPTS sets non-essential parameters for diamond alignment
+#
+# See the Diamond documentation available at GitHub:
+# https://github.com/bbuchfink/diamond. 
+#
+# Look particularly at options for sensitivity, scoring and memory and
+# performance.
+#
+# To set options, override this by inserting a row similar to the one below but
+# with whatever options you'd like to add in your Makefile *after* you include
+# this file.
+DIAMOND_ALIGN_OPTS =  
+
+# Output format of the DIAMOND results. The default value refers to a tabular file.
+# For more format options see https://github.com/bbuchfink/diamond.
+OUTFMT = 6
+
+# Download HTTP URL for the BacMet protein database
+# In case you are not using ftp, you can override this macro and paste here the https URL.
+HTTP = http://bacmet.biomedicine.gu.se/download/BacMet2_EXP_database.fasta
+# IMPORTANT: databases are updated recurrently, so always check the availability of the http
+
+# Download HTTPS URL for the bacmet.py script, necessary for the downstream analyses
+HTTPS = https://gist.github.com/08ab264f0ea127cc242d99bc3844c743.git 
```

Comment:
It would be better if you could package your script in Conda and instruct users to install that way. See e.g. https://github.com/erikrikarddaniel/hmmrank

Reply:
Agreed, but I am short on time right now. Can we make the github gists do for the time being, please?

Reply:
Fine, but I'd prefer if the instructions are to put it in `~/bin` rather than in the working directory.